chef/cookbooks/network/recipes/default.rb

```diff
@@ -638,6 +638,14 @@ def get_datapath_id_for_ovsbridge(bridge)
     code "wicked ifup all"
     only_if { run_wicked_ifup }
   end
+  # Sometimes wicked deletes the default route upon bringing up a new interface,
+  # make sure it gets put back
+  unless ::Kernel.system("ip route |grep -q default")
+    nic = default_route[:nic]
+    gw = default_route[:gateway]
+    Chef::Log.info("Adding default route via #{gw} to #{nic}")
+    ::Kernel.system("ip route add default via #{gw} dev #{nic}")
+  end
```

Comment:
Do I understand this correctly that this is a workaround while we see this issue with the default route being dropped? I can't quite tell from the bug report, do you have an inkling as to what is ultimately the cause?